pages/contact.vue

```diff
@@ -1,15 +1,109 @@
 <template>
-  <div class="white-wrapper">
-    Contact Page
+  <div class="background-section">
+    <div class="title">
+      <h1 class="cursive">Contact & Directions</h1>
+      <h4><font-awesome-icon icon="phone"/> (617) 924-9595</h4>
+    </div>
+    <div class="contact-section">
+        <div class="outside-image">
+          <img src="outside2.jpg">
+        </div>
+        <div class="store-info-section">
+          <contact-info />
+        </div>
+    </div>
+    <div class="call-section">
+      <div class="order">
+        <a :href="`tel:${6179249595}`">
+          <div class='phone-number'> 
+            <font-awesome-icon icon="phone"/> (617) 924-9595 
```

Comment:
If we just give this icon a `class` of "phone-number" then I don't think we'll need the div surrounding it.

Reply:
You are correct, removed the div and added the phone-number class to the anchor tag for same result.